### DISCUSSION TITLE
How do I get people to join my organization?

### DISCUSSION THREAD

**Discussion by xItzDiamondx** [ScratchAddons/ScratchAddons#2134] [Help] [answered]
So, ScratchAddons has grown a lot, and I've started an organization with another developer. I wanted to ask how you all got such great developers on the team?

**Comment by zoraezimran** [comment]
yeah I'm the leader and we need more people to join our organization and help develop new things with code

**Reply by zoraezimran** [reply]
or is it possible for the people from your organization to also join ours

**Reply by zoraezimran** [reply]
https://github.com/ZPSD-Products

**Reply by WorldLanguages** [reply]
I'd recommend you make an offline capable PWA instead of .exe files, which only work on Windows and are sketchy bc they can be malicious

**Reply by zoraezimran** [reply]
ok but can you get people to join our organization or can you join

**Reply by zoraezimran** [reply]
pls

**Comment by GrahamSH-LLK** [comment]
Scratch Addons is 
A. Popular: we replaced SME, so we started with tons of users. Plus, our user base is mainly coders
B. Lucky: I don’t know exactly how we got so big, but people who contributed a lot got added!

**Reply by xItzDiamondx** [reply]
Hrm... Do you know a few people who might want to join?

**Reply by WorldLanguages** [reply]
@xItzDiamondx It's hard to tell when your org doesn't seem to have any specific purpose

**Reply by xItzDiamondx** [reply]
True

**Reply by xItzDiamondx** [reply]
Oh, also, I was wanting to ask, I wanna help out, but I don't KNOW what I can help with.

**Reply by GrahamSH-LLK** [reply]
> Oh, also, I was wanting to ask, I wanna help out, but I don't KNOW what I can help with.

Any issues that you think you can tackle, feel free to try!

**Reply by xItzDiamondx** [reply]
What languages do I need to know? I think Git and json?

**Reply by GrahamSH-LLK** [reply]
> What languages do I need to know? I think Git and json?

Git is a tool to push your code, JSON is for addon files, and most of the extension is written in JavaScript.

**Reply by xItzDiamondx** [reply]
I know Javascript pretty well

**Reply by xItzDiamondx** [reply]
So I guess I can start developing!

**Reply by xItzDiamondx** [reply]
But, as a begginer, what do u think i should try out first?

**Reply by GrahamSH-LLK** [reply]
> But, as a begginer, what do u think i should try out first?

Pick anything

**Reply by xItzDiamondx** [reply]
Hrm.... WDYT

**Reply by xItzDiamondx** [reply]
Like an easy project

**Reply by xItzDiamondx** [reply]
@WorldLanguages How do I come up with a good organization name

**Comment by xItzDiamondx** [comment]
You there @GrahamSH

**Comment by GrahamSH-LLK** [comment, marked as answer]
This is pretty unrelated to SA. Contact us elsewhere if you want to talk about stuff like this.
